CHANGES.md

```diff
@@ -1,8 +1,76 @@
 # Change Log / Release Notes
 
-## 0.13
+## 0.13 (Sometime Early 2017)
 
-  * Re-builds using script `<xmonaddir>/build` if it exists and is executable
+### Breaking Changes
+
+  * When restarting xmonad, resume state is no longer passed to the
+    next process via the command line.  Instead, a temporary state
+    file is created and xmonad's state is serialized to that file.
+
+    When upgrading to 0.13 from a previous version, the `--resume`
+    command line option will automatically migrate to a state file.
+
+    This fixes issue #12.
+
+### Enhancements
+
+  * You can now control which directory xmonad uses for finding your
+    configuration file and which one is used for storing the compiled
+    version of your configuration.  In order of preference:
+
+      1. New environment variables.  If you want to use these ensure
+         you set the correct environment variable and also create the
+         directory it references:
+
+         - `XMONAD_CONFIG_DIR`
+         - `XMONAD_CACHE_DIR`
+         - `XMONAD_DATA_DIR`
+
+      2. XDG Base Directory Specification directories, if they exist:
+
+         - `XDG_CONFIG_HOME/xmonad`
+         - `XDG_CACHE_HOME/xmonad`
+         - `XDG_DATA_HOME/xmonad`
+
+      3. The `~/.xmonad` directory.
+
+    If none of these directories exist then (1) will be used if it is
```

Comment:
Oh, I think I understand now, you mean if none of them exist then (1) *will be created* if it is set, otherwise (2) will be created?  I think the text ought to mention something about directories being created.